lib/deaggregate.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+/*!
+ * Code adpted from aws-kinesis-agg 4.0.4
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const ProtoBuf = require('protobufjs');
+const aggJson = require('./aggregate-protobuf.json');
+
+const MAGIC_NUMBER = 'f3899ac2';
```

Comment:
I checked the documentation, and absolutely hate that they call it MAGIC_NUMBER. Can we rename the variable KPL_MAGIC_NUMBER, so its more clear where it comes from.